builtin/logical/transit/path_hmac_test.go

```diff
@@ -36,9 +37,9 @@ func TestTransit_HMAC(t *testing.T) {
 	}
 	// We don't care as we're the only one using this
 	lock.RUnlock()
-	keyEntry := p.Keys[p.LatestVersion]
+	keyEntry := p.Keys[strconv.Itoa(p.LatestVersion)]
```

Comment:
Set `lastestVersion := strconv.Itoa(p.LatestVersion)` so you don't have to call it multiple times.

Reply:
I'd say this applies anywhere where the converted `latestVersion` is used more than once.